ownCloud/UI Elements/LockWindow.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  LockWindow.swift
+//  ownCloud
+//
+//  Created by Javier Gonzalez on 17/05/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2018, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+
+//Subclass that only allows the protrait mode
+class LockWindow: UIWindow {
+
+    /*
+    // Only override draw() if you perform custom drawing.
+    // An empty implementation adversely affects performance during animation.
+    override func draw(_ rect: CGRect) {
+        // Drawing code
+    }
+    */
+
+    // MARK: - Show and hide animations
+    func showWindowAnimation(completion: (() -> Void)? = nil) {
```

Comment:
Those methods what make it is animate the Window to be shown or to be hide. I thinks that
`showWindowAnimation` and `hideWindowAnimation` are the best name and also because if you do not want to animate them you do not have to call those methods.